modules/libcom/src/osi/os/WIN32/osdThread.c

```diff
@@ -12,6 +12,12 @@
  * Author: Jeff Hill
  */
 
+#ifndef _WIN32_WINNT
+#   define _WIN32_WINNT 0x0600
+#elif _WIN32_WINNT < 0x0600
+#   error Minimum supported is Windows Vista
+#endif
```

Comment:
Also worth highlighting this change to explicitly require >= Vista.  I'm not certain what the current (implied) requirement is, so this may not be a practical change.